base/applications/fontview/fontview.c

```diff
@@ -72,27 +72,39 @@ FormatString(
 	return 0;
 }
 
-static void
-ErrorMsgBox(HWND hParent, DWORD dwMessageId, ...)
+static void FormatMsgBox(HWND hParent, DWORD dwMessageId, DWORD dwCaptionId, UINT uType, va_list args)
 {
 	HLOCAL hMemCaption = NULL;
 	HLOCAL hMemText = NULL;
-	va_list args;
 
-	va_start(args, dwMessageId);
 	FormatString(FORMAT_MESSAGE_ALLOCATE_BUFFER,
 	              NULL, dwMessageId, 0, (LPWSTR)&hMemText, 0, &args);
-	va_end(args);
-
 	FormatString(FORMAT_MESSAGE_ALLOCATE_BUFFER,
-	              NULL, IDS_ERROR, 0, (LPWSTR)&hMemCaption, 0, NULL);
-
-	MessageBoxW(hParent, hMemText, hMemCaption, MB_ICONERROR);
+	              NULL, dwCaptionId, 0, (LPWSTR)&hMemCaption, 0, NULL);
 
+	MessageBoxW(hParent, hMemText, hMemCaption, uType);
 	LocalFree(hMemCaption);
 	LocalFree(hMemText);
```

Comment:
Please fix indentation in that function as well (4-space indentation, instead of tabs).

Reply:
This code is already having mixed indentation, I think it would be better to reformat it separately.

Reply:
I have committed formatting fixes into master separately, and resolved it there. Should be good now.